src/modules/site-v2/templates/admin/etl/list.html

```diff
@@ -86,7 +86,11 @@
               </a>
             </td>
             <td class='text-center'>
+              {% if ops['id'] %}
```

Comment:
consider replacing the square brackets in the `if` with .get('id'). [] will throw if the key doens't exist. .get() will return default value None (safer).